examples/python/aws-eks/main.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+# for CDK
+from constructs import Construct
+from cdktf import App, TerraformStack, TerraformOutput, Token
+
+# for terraform provider
+from imports.aws import AwsProvider, DataAwsCallerIdentity
+from imports.local import LocalProvider
+from imports.null import NullProvider
+from imports.random import RandomProvider
+from imports.template import TemplateProvider
+
+# for terraform module
+from imports.terraform_aws_modules.vpc.aws import Vpc
+from imports.terraform_aws_modules.eks.aws import Eks
+
+class MyStack(TerraformStack):
+    def __init__(self, scope: Construct, ns: str):
+        super().__init__(scope, ns)
+
+        AwsProvider(self, 'Aws', region='us-west-2')
+        LocalProvider(self, 'Local')
+        NullProvider(self, 'Null')
+        RandomProvider(self, 'Random')
+        TemplateProvider(self, 'Template')
```

Comment:
Perhaps this is more of a Terraform question and less of a cdktf question, but do the last four providers do anything?

Reply:
Hm, not sure - I think these providers would be pulled in automatically if needed - aren't they?

Reply:
Hi @cmclaughlin and @skorfmann,

Thank for your reviewing,
Yes, the four providers not real take any effect during develop cdk for terraform,
and the provider will be downloaded when user perform `terraform init`,
hence, I have removed the four provider declaration and including from the example.